### PR TITLE
Add ExecuTorch warning to mobile_optimizer

### PR DESCRIPTION
**Pull request by svekars**
Preview: https://docs-preview.pytorch.org/pytorch/pytorch/134697/mobile_optimizer.html

cc @brycebortree @tstatler

### CODE REVIEW

**Review by ali-khosh — APPROVED** [review]
Reviewed changes to highlight PT Mobile is deprecated.

File: docs/source/mobile_optimizer.rst [hunk 1]
```diff
@@ -2,7 +2,11 @@ torch.utils.mobile_optimizer
 ===================================
 
 .. warning::
-    This API is in beta and may change in the near future.
+   PyTorch Mobile is no longer actively supported. Please check out
+   `ExecuTorch <https://pytorch.org/executorch-overview>`__, PyTorch’s
```
Comment (malfet) [inline docs/source/mobile_optimizer.rst:6]: Please don't use unicode symbols unless necessary
```suggestion
   `ExecuTorch <https://pytorch.org/executorch-overview>`__, PyTorch's
```

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**